src/messageschannel.hpp

```diff
@@ -48,6 +49,7 @@ class JoinRoomJob;
 class UploadContentJob;
 class GetContentJob;
 class DownloadFileJob;
+class SetTypingJob;
```

Comment:
Not used?

Reply:
If I'm not mistaken, I'm using it at `MatrixMessagesChannel::sendTypingNotification`: `callApi<QMatrixClient::SetTypingJob>`

Reply:
oh, my bad, you're right